src/source/Crypto/Dtls_mbedtls.c

```diff
@@ -196,6 +201,7 @@ STATUS dtlsTransmissionTimerCallback(UINT32 timerID, UINT64 currentTime, UINT64
     handshakeStatus = mbedtls_ssl_handshake(&pDtlsSession->sslCtx);
     switch (handshakeStatus) {
         case 0:
+        // success. 
```

Comment:
Make sure you run the clang format validator check-clang.sh. If this fails, then you have a format issue and you can correct it by running clang-format.sh -f src/* (check please). This will fix the file that has a formatting issue

Reply:
Sure, I am aware of the clang-format error in the previous commit, but I got several syntax error in the yaml. Does this sdk prefer the specific version of clang-format?

Reply:
Not sure what you mean, the check-clang.sh should run clean on the existing codebase.